walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -64,6 +64,29 @@ else if (request.getDuration() > TimeRange.WHOLE_DAY.duration()){
 
 
         for (Event event2 : events){
+
+            //optional attendee in every attendee considered 
```

Comment:
My overall logic is flawed in this code (I just did what I could) but here it is:
1. Check the 2 events of the attendees. If they don't overlap and have sufficient time between them, put the meeting before, in-between, or after.
2. Then, compare event3 to the existing temporary availibility. Once I get to this point I get a bit confused:
- if there is times that work for everyone with this optional attendees 3rd event, then remove all the times from temp_availability that don't work 
-if there are no times that work for the optional attendee, ignore them and their event